test/lib/Elastica/Test/ClientTest.php

```diff
@@ -26,6 +26,16 @@ public function testConstruct()
     /**
      * @group functional
      */
+    public function testGetVersion()
+    {
+        $client = $this->_getClient();
+        $this->assertNotEmpty($version = $client->getVersion());
+        $this->assertSame($version, $client->getVersion());
```

Comment:
I've added new ENV to Dockerfile and then checking it in test, see 0abec0e2961135d17a6d1ae87ef07c07683a5524